### PR TITLE
Install and usage example for Minio.

### PR DESCRIPTION
**Pull request by hackintoshrao**
- Adding the example as suggested in the review for https://github.com/mesosphere/universe/pull/840 . 
- cc @ryadav88

### CODE REVIEW

File: 1.8/minio/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+The instructions below use a pinned hostname constraint to ensure the application is always restarted on the same host by Marathon. This allows it to get back to its data but means that you could lose data if that agent goes down. 
+
+- Estimated time for completion: up to 5 minutes. 
+- Target audience:
+ - Operators
+ - Application admins
+ - Developers 
+ - Devops Engineers
+- Scope: Learn to install Minio on DCOS and learn to use it using minio command line tool `mc`.
```
Comment (mhausenblas) [inline 1.8/minio/README.md]: DCOS -> DC/OS

File: 1.8/minio/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+- IP address of the public agent(s) where Marathon-lb or an available hostname configured to point to the public agent(s) where Marathon-lb is running.
+
+## Setting up Minio 
+
+1. Before starting, identify the IP address or hostname of a public agent where Marathon load balancer is running. 
+
+
+- Visit the Universe page in DC/OS, and click on the "Install Package" button underneath Minio.
+
+! [Install Minio](img/install.png)
```
Comment (mhausenblas) [inline 1.8/minio/README.md]: Remove space between `!` and `[` so that the image renders correctly

File: 1.8/minio/README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+- Click on "Advanced Installation" and navigate to the "networking" tab. Specify the IP address or hostname of the public agent where marathon load balancer is running. Make sure you remove the leading http:// and the trailing / from the IP. 
+
+! [Configure IP](img/ip.png)
```
Comment (mhausenblas) [inline 1.8/minio/README.md]: Remove space between `!` and `[` so that the image renders correctly

File: 1.8/minio/README.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+## Setting up Minio 
+
+1. Before starting, identify the IP address or hostname of a public agent where Marathon load balancer is running. 
+
+
+- Visit the Universe page in DC/OS, and click on the "Install Package" button underneath Minio.
+
+! [Install Minio](img/install.png)
+
+- Click on "Advanced Installation" and navigate to the "networking" tab. Specify the IP address or hostname of the public agent where marathon load balancer is running. Make sure you remove the leading http:// and the trailing / from the IP.
```
Comment (mhausenblas) [inline 1.8/minio/README.md]: Please make sure that you're using Marathon-LB consistently (or shorten to MLB).

**Comment by mhausenblas** [comment]
Thanks a lot  @hackintoshrao and now waiting for @ryadav88 to merge your Universe PR and then I merge this here.

**Comment by ryadav88** [comment]
@mhausenblas Universe PR is merged. This is ready to merge.

**Comment by ryadav88** [comment]
@hackintoshrao Once this example is merged, can you add the following line at the end of `preInstallNotes` in Universe: `\n\nCheck the service tutorial at https://github.com/dcos/examples/tree/master/1.8/minio`

**Pull request merged**